src/crimson/osd/recovery_backend.cc

```diff
@@ -60,6 +60,82 @@ void RecoveryBackend::WaitForObjectRecovery::stop() {
   }
 }
 
+void RecoveryBackend::handle_backfill_finish(
+  MOSDPGBackfill& m)
+{
+  logger().debug("{}",
```

Comment:
nit, i'd suggest avoid wrapping lines proactively.

Reply:
Changed in entire `recovery_backend.cc`.